packages/dts-plugin/src/plugins/GenerateTypesPlugin.ts

```diff
@@ -11,6 +11,7 @@ import {
   generateTypesInChildProcess,
   retrieveTypesAssetsInfo,
 } from '../core/index';
+import path from 'path';
```

Comment:
```suggestion
import path from 'node:path';
```

Reply:
why node:path over path? 
What range is it supported on?

Reply:
To make it compatible with runtimes different than NodeJS 

It is supported since the current lts (v18)

Reply:
I need to retain node 14 support. 18 isn't common in a lot of enterprise users.